packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -1,22 +1,20 @@
 import useRealTotalAssetCurrency from './useTotalAssetCurrency';
 import useExchangeRate from './useExchangeRate';
-
 /**
  * we can use this hook to get the total balance of the given accounts list.
  * it loops through the accounts list and adds the balance of each account
  * to the total balance, it also converts the balance to the currency of the
  * first account in the list
  */
-
-const useTotalAccountBalance = (accounts: { balance?: number; currency?: string }[]) => {
+const useTotalAccountBalance = (accounts: { balance?: number; currency?: string; account_type?: string }[]) => {
     const total_assets_real_currency = useRealTotalAssetCurrency();
     const { getRate } = useExchangeRate();
 
     if (!accounts.length) return { balance: 0, currency: total_assets_real_currency };
 
     const balance = accounts.reduce((total, account) => {
-        const base_rate = getRate(total_assets_real_currency || '');
-        const rate = getRate(account.currency || total_assets_real_currency || '');
```

Comment:
please make sure this doesn't break anything. because I don't think we need to remove it :pray: